### PR TITLE
Add Scrap Latest Torrents

### LINKED ISSUE #165: Scrap Latest Torrents
# Description

Create a Script that can scrap the latest torrents as per the labels given.

## Language

- [X] Python
- [X] Golang

## Type of issue

- [X] Feature

## Checklist:

- **For Feature**

|name|about|labels|assignees|
|----|-----|------|---------|
|Scrap Latest Torrents|Get Latest Torrents straight on your CLI as per the labels given|PSoC-2020, Python, Golang, Hard|`Open for PSoC-2020 Participants`|

### PR DESCRIPTION
**Pull request by DiegoCaraballo**
# Description

This script downloads the torrents in our Torrent client from the website https://rarbg.to/ from an entered label.

Fixes #165

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Scrap_Latest_Torrents/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+1.  Create a Virtual Environment.
+2.  Install the requirements by using  `pip3 install -r requirements.txt`
+
+### Running a File
+
+1.  Run the Script  `python scrap_latest_torrents.py`
+2.  Enter the label to search, example: Mr Robot.
+3.  The number of Torrents found will be displayed, you must confirm the download
+
+![How to use the script](https://i.ibb.co/PD3sPLf/Scrap-Latest-Torrents.png)
```
Comment (vybhav72954) [inline Python/Scrap_Latest_Torrents/README.md:20]: Hey, can you add an Author Subheading as well, we would love to have your name here.

File: Python/Scrap_Latest_Torrents/scrap_latest_torrents.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    Function that opens the torrent file according to the operating system
+    """
+    if sys.platform == "win32":
+        os.startfile(filename)
+    else:
+        opener = "open" if sys.platform == "darwin" else "xdg-open"
+        subprocess.call([opener, filename])
+
+if __name__ == "__main__":
+    main()
```
Comment (vybhav72954) [inline Python/Scrap_Latest_Torrents/scrap_latest_torrents.py:64]: Please add a blank line at EOF

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
Some minor changes

File: Python/Scrap_Latest_Torrents/README.md (hunk 1 above)
Comment (DiegoCaraballo) [reply Python/Scrap_Latest_Torrents/README.md:20]: Add Author

File: Python/Scrap_Latest_Torrents/scrap_latest_torrents.py (hunk 2 above)
Comment (DiegoCaraballo) [reply Python/Scrap_Latest_Torrents/scrap_latest_torrents.py:67]: @vybhav72954 It's weird, locally I see the line at the end.

**Review by vybhav72954 — APPROVED** [review]
@HarshCasper @sohamsshah 
This looks good to me

File: Python/Scrap_Latest_Torrents/scrap_latest_torrents.py (hunk 2 above)
Comment (vybhav72954) [reply Python/Scrap_Latest_Torrents/scrap_latest_torrents.py:67]: interesting, maybe one extra whiteline 🤔

Comment (DiegoCaraballo) [reply Python/Scrap_Latest_Torrents/scrap_latest_torrents.py:67]: @vybhav72954 Do you need me to change it?

File: Python/Scrap_Latest_Torrents/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+## Scrap Latest Torrents
+
+This script downloads the torrents in our Torrent client from the website https://rarbg.to/ from an entered label.
+
+### Requirements
+
+1.  You must have a torrent client, for example: uTorrent, BitTorrent, etc
```
Comment (sohamsshah) [inline Python/Scrap_Latest_Torrents/README.md:7]: You can provide links to download uTorrent and BitTorrent here itself. Adding a hyperlink would be good IMO

File: Python/Scrap_Latest_Torrents/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+### Setup
+
+1.  Create a Virtual Environment.
```
Comment (sohamsshah) [inline Python/Scrap_Latest_Torrents/README.md:11]: Can you brief about commands to create a virtual env in python? Would help beginners who are using this script.

File: Python/Scrap_Latest_Torrents/scrap_latest_torrents.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    """
+    count = 0
+
+    # Label to search
+    label = input("Enter a label: ")
+
+    print("Searching torrents... please wait")
+
+    # API RARGB
+    client = rarbgapi.RarbgAPI()
```
Comment (sohamsshah) [inline Python/Scrap_Latest_Torrents/scrap_latest_torrents.py:24]: You can abstract the API call into another UDF. So it becomes more readable and flexbile.

**Review by sohamsshah — CHANGES_REQUESTED** [review]
This looks good to me and also it has a good utilitarian value to the project. Thanks for working on this! 

I have suggested small enhancements to your script. After these changes, this is good to go from my side.

File: Python/Scrap_Latest_Torrents/README.md (hunk 3 above)
Comment (DiegoCaraballo) [reply Python/Scrap_Latest_Torrents/README.md:7]: I add links

File: Python/Scrap_Latest_Torrents/README.md (hunk 4 above)
Comment (DiegoCaraballo) [reply Python/Scrap_Latest_Torrents/README.md:13]: I add instructions

File: Python/Scrap_Latest_Torrents/scrap_latest_torrents.py (hunk 5 above)
Comment (DiegoCaraballo) [reply Python/Scrap_Latest_Torrents/scrap_latest_torrents.py]: I add a new function

**Review by HarshCasper — APPROVED** [review]

**Comment by HarshCasper** [comment]
Thanks @DiegoCaraballo 🎉

**Pull request merged**